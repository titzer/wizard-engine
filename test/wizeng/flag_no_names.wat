(module
  (func (export "main")
    (call $start)
  )
  (func $start (export "_start")
    (call $f0)
  )
  (func $f0
    unreachable
  )
)
