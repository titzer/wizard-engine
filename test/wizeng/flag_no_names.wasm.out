<wasm func #0> +1
  <wasm func #1> +1
    <wasm func #2> +1
      !trap[UNREACHABLE]
